bot.py

```diff
@@ -50,6 +52,21 @@
     else:
         await client.send_message(message.channel, 'The current song is : "{}"'.format(elevator.title))
 
+async def playSound(message, soundURL):
+    if not in_elevator:
+        voice_channel = client.get_channel(message.author.voice.voice_channel.id)
+        voice_client = await client.join_voice_channel(voice_channel)
+        genji = await voice_client.create_ytdl_player(soundURL)
+        genji.volume = .2
+        genji.start() 
+        genji.sleep(5) #idk if theres a "wait for playback finish"
```

Comment:
Sleep is a generic python lib function. The comment explains its usage. I wasn't aware that the player was an officially supported Discord feature. I thought it was some external library. In this case, yes, we should use (while !genji.is_done())